### ISSUE TITLE
Audio link issue on Improve Accessibility of Audio Content with the Audio Element challenge, Beta

### ISSUE THREAD

**Issue by Skarrix** [freeCodeCamp/freeCodeCamp#13879]
Challenge [improve-accessibility-of-audio-content-with-the-audio-element](http://beta.freecodecamp.com/en/challenges/applied-accessibility/improve-accessibility-of-audio-content-with-the-audio-element) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

Challenge works fine but there may be an issue with the provided sound clip, which plays garbled and jumbled audio as if it is set to play too fast. Plays that way both through the preview screen as well as when visiting the link separately in Incognito mode.

```html


<body>
  <header>
    <h1>Real Coding Ninjas</h1>
  </header>
  <main>
    <p>A sound clip of Zersiax's screen reader in action.</p>
  <audio controls>
<source src="https://s3.amazonaws.com/freecodecamp/screen-reader.mp3" type="audio/mpeg" /> 
    </audio>  
    
  </main>
</body>

```

**Comment by HKuz** [comment]
@Skarrix - thanks for raising this issue. Amazingly, that's how fast screen reader users set their devices to read and is what the sound clip is supposed to sound like. If you're interested, here's [a link to an article](https://medium.freecodecamp.com/looking-back-to-what-started-it-all-731ef5424aec#.ryenxo5fb) written by @zersiax with more info about how he codes 😄 and [a link to the original Soundcloud clip](https://soundcloud.com/freecodecamp/zersiaxs-screen-reader). 

To avoid confusion in the future, I think we can add a note at the very bottom of the instructions to that says something like:

<strong>Note:</strong><br>The audio clip may sound fast and be difficult to understand, but that is a normal speed for screen reader users.

I'll mark this as "Help Wanted" - for  anyone who wants to make this change, please refer to the [Contributor's Guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md) and ask any questions in the [Contributor's chat room](https://gitter.im/FreeCodeCamp/Contributors). Code for this challenge can be found [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/seed/challenges/01-responsive-web-design/applied-accessibility.json#L386)

**Comment by zersiax** [comment]
This issue amuses me :) Basically what you are hearing is a screen reader reading out the first part of a prior version of the article you just received a link to, however, it is set to a rate of about 700 words per minute, which is hard to follow for the untrained ear :) It is in fact not garbled, I can write the transcription for you word for word if you like ;)

**Comment by Skarrix** [comment]
:o I had no idea! That is quite amazing and very interesting.

Thanks for the clarification and the info links @HKuz and @zersiax 
I have no idea how you keep pace with that @zersiax, that's one hell of a skill!

**Comment by cdrainxv** [comment]
I'll take this up.

**Comment by HKuz** [comment]
@Skarrix , @zersiax - my untrained ear can't follow it either, that's like a super power to me 😄 

Thanks for taking this up @cdrainxv 👍

**Comment by Newcoder888** [comment]
I can't seem to solve this problem. Can anyone help please? 
Here is my code 
<body>
  <header>
    <h1>Real Coding Ninjas</h1>
  </header>
  <main>
    <p>A sound clip of Zersiax's screen reader in action.</p> 
  <audio controls>
  <source src="https://s3.amazonaws.com/freecodecamp/screen-reader.mp3" type="audio/mpeg">
/audio> 

I click run, and code camp comes back with....cannot read property 'length'  of null???

**Comment by raisedadead** [comment]
Please search through open issues, and open a fresh one only if you find none that relates to your issue. Alternatively seek assistance on our [Forum](https://www.freecodecamp.org/forum)
